pkgs/top-level/all-packages.nix

```diff
@@ -23224,7 +23224,7 @@ with pkgs;
 
     # latest 6.8 mysteriously fails to parse '@headings single':
     #   https://lists.gnu.org/archive/html/bug-texinfo/2021-09/msg00011.html
-    texinfo = texinfo6_7;
+    texinfo = buildPackages.texinfo6_7;
```

Comment:
I wonder why this is needed. `gpm` seems to put `texinfo` into `nativeBuildInputs`. I would expect it to be enough.

Reply:
https://github.com/NixOS/nixpkgs/issues/49526